pixi.js.d.ts

```diff
@@ -393,8 +393,14 @@ declare namespace PIXI {
         protected _recursivePostUpdateTransform(): void;
         getBounds(skipUpdate?: boolean, rect?: Rectangle): Rectangle;
         getLocalBounds(rect?: Rectangle): Rectangle;
-        toGlobal(position: Point, point?: Point, skipUpdate?: boolean): Point;
-        toLocal(position: Point, from?: DisplayObject, point?: Point, skipUpdate?: boolean): Point;
+        //creates and returns a new point
+        toGlobal(position: Point | ObservablePoint): Point;
```

Comment:
In #156 @clark-stevenson thought it would be more clear to use Point|ObservablePoint instead of the PointLike interface.